fala/apps/adviser/tests/test_pagination_playwright.py

```diff
@@ -0,0 +1,19 @@
+from playwright.sync_api import expect
+from fala.playwright.setup import PlaywrightTestSetup
+
+
+class PaginationResults(PlaywrightTestSetup):
+
+    def test_results_pagination(self):
+        checkboxes = ["Crime"]
+
+        page = self.visit_results_page("M25 3JF", checkboxes)
+        expect(page.item_from_text("1576 results in order of closeness to M25 3JF.")).to_be_visible()
+        expect(page.pagination_link_title).to_be_visible()
+        page.next_link.click()
+        # this tests that changing page persists the category choice by not affecting the result count
+        expect(page.item_from_text("1576 results in order of closeness to M25 3JF.")).to_be_visible()
```

Comment:
oh I see why you have done this now with the number of results - still I'm not too sure this is the best way to test in case that number changes. Is it possible to target the first provider card and make sure the category `crime` is in the list for it?

Reply:
Because that's what we really want to know, rather than inferring it from the count

Reply:
the count way makes it work across the entire search. We could check that a specific result on a page has the correct category, but we cannot safely extrapolate to all providers on that page. The count ensures that the basics of the search (category and postcode) do not change between pages.

Reply:
okay i see what you mean - if there really is no other way to reliably check the categories have persisted then I guess this is fine at the min